### DISCUSSION TITLE
Monday 20th december

### DISCUSSION THREAD

**Discussion by ap1987-ap** [munoztd0/DAP_2021#14] [General]
Hy David

I was wondering if the meeting next monday also is via zoom. 
If yes, i wouldnt have to travel to geneve. If not that is ok too.. I just like to know before i have to take the train ;-)

Thank you for your anwer
Best Anja

**Comment by munoztd0** [comment]
Hello,

I will be there on site but we can make it work on zoom also if you want/need to.
